src/config/locales.md

```diff
@@ -1,6 +1,8 @@
 # Locales and Translations
 
-For a list of currently enabled locales, run
+You can use the following command, which is specific to `glibc` installations of
+Void (given that `musl` does not support locales), to see a list of currently
+enabled locales:
```

Comment:
personally, I think this is better as an introductory sentence/paragraph, so something like this

```suggestion
glibc supports setting the system locale, while musl does not. Both support setting the language for applications.

## Listing locales

For a list of currently enabled locales, run:
```

Reply:
I like that too, fixed.